python/cudf/dataframe/series.py

```diff
@@ -700,6 +700,16 @@ def masked_assign(self, value, mask):
         data = self._column.masked_assign(value, mask)
         return self._copy_construct(data=data)
 
+    def dropna(self):
+        """
+        Return a Series with null values removed.
+        """
+        if self.null_count == 0:
+            return self
+        data = self._column.dropna()
+        index = self.index.loc[~self.isna()]
```

Comment:
This is inefficient but we'll have to wait for the multi-column dropna to do better